src/commands/run-tests-with-ats.yaml

```diff
@@ -0,0 +1,35 @@
+parameters:
+  chart_archive_prefix:
+    description: "Prefix for the chart archive file to execute tests for."
+    type: string
+    default: ""
+  app-test-suite_version:
+    description: "Version of app-test-suite dats.sh container wrapper to use (git tag or commit)"
+    type: string
+  app-test-suite_container_tag:
+    description: "Container tag of app-test-suite to use (check quay.io/giantswarm/app-test-suite)"
+    type: string
+  additional_app-test-suite_flags:
+    description: "Additional app-test-suite flags to use"
+    type: string
+    default: ""
+steps:
+  - attach_workspace:
+      at: build
+  - run:
+      name: "download dats.sh script"
+      command: |
+        wget "https://raw.githubusercontent.com/giantswarm/app-test-suite/<< parameters.app-test-suite_version >>/dats.sh"
+        chmod +x dats.sh
+  - run:
+      name: "execute app-test-suite with dats.sh wrapper"
+      command: |
+        export DATS_TAG=<< parameters.app-test-suite_container_tag >>
+        # find the chart archive file name
+        chart_archive=$(find build -name "<< parameters.chart_archive_prefix >>*.tgz" -print -quit)
+        # move chart archive from build directory to root
+        # we're doing this to make sure .ats/main.yaml is picked up
+        cp "$chart_archive" "${chart_archive##*/}"
+
+        # execute dats wrapper
+        ./dats.sh --chart-file "${chart_archive##*/}" << parameters.additional_app-test-suite_flags >>
```

Comment:
Slightly off topic but something I have been wondering:
Can we reduce the permissions / settings required for `dats.sh`? AFAIK it will still not work like this with e.g. podman.

Reply:
Created https://github.com/giantswarm/giantswarm/issues/18884

Feel free to add your observations in the issue!